src/Common/Commands.Common/AzureDataCmdlet.cs

```diff
@@ -111,48 +111,17 @@ protected override void SaveDataCollectionProfile()
             WriteWarning(string.Format(Resources.DataCollectionSaveFileInformation, fileFullPath));
         }
 
-        protected override void PromptForDataCollectionProfileIfNotExists()
+        protected override void SetDataCollectionProfileIfNotExists()
         {
-            // Initialize it from the environment variable or profile file.
             InitializeDataCollectionProfile();
 
-            if (!_dataCollectionProfile.EnableAzureDataCollection.HasValue && CheckIfInteractive())
```

Comment:
@shahabhijeet, the change itself is not _huge_, per se.  As far as I can tell, there exists no tests currently.

I would say the scenarios would be:
1. Upon clean install, warning is printed on first run and `_dataCollectionProfile.EnableAzureDataCollection` is properly set to `true` in the user's context/profile.
2. Upon update, this method `return` a because `_dataCollectionProfile.EnableAzureDataCollection.HasValue` is `true` (i.e., existing user settings with respect to data collection are preserved).
3. Upon clean install, any call to this method with the context of a script would return because `CheckIfInteractive()` is `false`.

How are startup/context scenarios normally tested?  I could unit test this method, but that seems about it.  Such a test would not really ensure (2).  I assume `CheckIfInteractive()` could be mocked to ensure (3)?